projects/packages/device-detection/src/class-device-detection.php

```diff
@@ -51,7 +51,8 @@ public static function get_info( $ua = '' ) {
 			/**
 			 * Filter the value of Device_Detection::get_info.
 			 *
-			 * @since  8.7.0
+			 * @since 1.0.0
+			 * @since-jetpack 8.7.0
```

Comment:
The conversion table shows that v1.0.0 of the device-detection package was released with Jetpack 8.7. Do we need this `@since-jetpack` annotation?

Reply:
you're right. we don't. Thanks for catching this. I'll fix it asap

Reply:
fixed